Digipost.Signature.Api.Client.Core/BaseClient.cs

```diff
@@ -69,18 +93,28 @@ private WebRequestHandler MutualTlsHandler()
             var certificateCollection = new X509Certificate2Collection {ClientConfiguration.Certificate};
             var mutualTlsHandler = new WebRequestHandler();
             mutualTlsHandler.ClientCertificates.AddRange(certificateCollection);
-            mutualTlsHandler.ServerCertificateValidationCallback = IsValidServerCertificate;
+            mutualTlsHandler.ServerCertificateValidationCallback = ValidateServerCertificateThrowIfInvalid;
 
             return mutualTlsHandler;
         }
 
-        private bool IsValidServerCertificate(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslpolicyerrors)
+        private bool ValidateServerCertificateThrowIfInvalid(object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslpolicyerrors)
         {
+            if (!ClientConfiguration.CertificateValidationPreferences.ValidateResponseCertificate)
+            {
+                Log.Warn("Validation of response certificate is disabled and should only be disabled under special circumstances. This validation is in place to ensure that the response is from the server you are expecting.");
```

Comment:
Tja, det er jo ikke så gæærnt egentlig, det er jo bare en konfigparameter for å _skru av_ en validering. Fiks feilen nå og ikke tenk mer på det 🙈